src/wirecraft/client/game.py

```diff
@@ -203,7 +225,7 @@ def handle_keydown(self, event: pygame.event.Event) -> None:
             else:
                 self.is_placing_cable = False
                 if self.cables:
-                    self.cables.pop()
+                    self.server.delete_cable(self.cables[-1].db_id)
```

Comment:
Cable is deleted on server side, but is it on client side as well ? (removed from self.cables)

Reply:
Cable list is wiped and fetched each frame, so not necessary (TODO use event soon :tm:)

Reply:
This is wild 🦁